apiserver/plane/db/migrations/0049_auto_20231116_0713.py

```diff
@@ -0,0 +1,72 @@
+# Generated by Django 4.2.5 on 2023-11-15 09:16
+
+# Python imports
+import uuid
+
+from django.db import migrations
+
+
+def update_pages(apps, schema_editor):
+    try:
+        Page = apps.get_model("db", "Page")
+        PageBlock = apps.get_model("db", "PageBlock")
+        PageLog = apps.get_model("db", "PageLog")
+
+        updated_pages = []
+        page_logs = []
+
+        # looping through all the pages
+        for page in Page.objects.all():
```

Comment:
use `values_list`

Reply:
You can't directly update dictionaries or tuples obtained from values() or values_list() and then perform a bulk update.